test/common/config/grpc_mux_impl_test.cc

```diff
@@ -159,18 +159,23 @@ TEST_F(GrpcMuxImplTest, ResetStream) {
   expectSendMessage("baz", {"z"}, "");
   grpc_mux_->start();
 
+  // Send another message for foo so that the node is cleared in the cached request.
+  expectSendMessage("foo", {"z", "x", "y"}, "");
```

Comment:
Regarding https://github.com/envoyproxy/envoy/pull/14080/files#r526407920

OK... I didn't realize this clear op did not exist.

Reply:
(added another comment line to clarify)